local_node/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+
+services:
+  api:
+    image: "neurobagel/api:${NB_API_TAG:-latest}"
+    ports:
+      - "${NB_API_PORT_HOST:-8000}:${NB_API_PORT:-8000}"
+    environment:
+      NB_GRAPH_USERNAME: ${NB_GRAPH_USERNAME}
+      NB_GRAPH_PASSWORD: ${NB_GRAPH_PASSWORD}
+      NB_GRAPH_ADDRESS: ${NB_GRAPH_ADDRESS:-graph}
+      NB_GRAPH_PORT: ${NB_GRAPH_PORT:-5820}
+      NB_GRAPH_DB: ${NB_GRAPH_DB:-test_data/query}
+      NB_RETURN_AGG: ${NB_RETURN_AGG:-true}
+      NB_API_PORT: ${NB_API_PORT:-8000}
+      NB_API_ALLOWED_ORIGINS: ${NB_API_ALLOWED_ORIGINS}
+  graph:
+    image: "${NB_GRAPH_IMG:-stardog/stardog:8.2.2-java11-preview}"
```

Comment:
We should use graphDB here, not stardog. Apart from us deprecating stardog, this will also not work anymore if they removed the tags.